packages/jellyfish-api-core/__tests__/category/loan/listAuctionHistory.test.ts

```diff
@@ -615,25 +615,25 @@ describe('Loan listAuctionHistory', () => {
           { index: 1 }
         )
 
-        expect(page.length).toStrictEqual(4)
+        expect(page.length).toStrictEqual(2)
```

Comment:
```
// This test shows that we can not use index only for the pagination start point as the DB composite index starts with maxBlockHeight.
```
so earlier, it did not work with `index` only and now it works? if then the comment should be removed.
Wondering what changed it. likewise does "should not listAuctionHistory with vaultId only" also changed?